packages/@aws-cdk/aws-rds/lib/props.ts

```diff
@@ -221,6 +228,7 @@ export abstract class Credentials {
     return {
       username: username ?? secret.secretValueFromJson('username').toString(),
       password: secret.secretValueFromJson('password'),
+      secretName: secret.secretName,
```

Comment:
Maybe Yes. I thought it may need to initialize and expose as a field like ‘encryptionKey’ below, so I wrote this line. 

https://github.com/hedrall/aws-cdk/blob/7e5ee80ea28e4e8d123a7862f887f0f727fc011b/packages/%40aws-cdk/aws-rds/lib/props.ts#L227-L235

I not sure of any use case that uses the secretName of the object of Credentials created by fromSecret, so I will delete it now.